docs/neopixel.rst

```diff
@@ -51,11 +51,17 @@ Classes
 
 .. py:class::
     NeoPixel(pin, n)
+    NeoPixel(pin, n, 4)
 
     Initialise a new strip of ``n`` number of neopixel LEDs controlled via pin
-    ``pin``. Each pixel is addressed by a position (starting from 0). Neopixels
-    are given RGB (red, green, blue) values between 0-255 as a tuple. For
-    example, ``(255,255,255)`` is white.
+    ``pin``. For **V2** micro:bit boards that support RGBW neopixels, a third
+    value can be passed to ``NeoPixel`` to indicate the number of bytes per
+    pixel is 4 rather than the default of 3.
```

Comment:
I think we can select an unspecified number of "pixels"? So while the main two options are 3 or 4 values, technically more could be sent if a new class of neopixel was released in the future.

I probably wouldn't mention it like that here as that could be confusing, but something along the lines of " number of bytes per pixel, where the default value of 3 is used for RGB and GRB, and 4 can be used for RGBW".

Reply:
Updated in https://github.com/bbcmicrobit/micropython/pull/694/commits/474f3cc7442f540d016ee946f14475666bcb78a6

Reply:
UPdated in https://github.com/bbcmicrobit/micropython/pull/694/commits/474f3cc7442f540d016ee946f14475666bcb78a6